### PR TITLE
fix: spelling and Markdown linter issues in the CoC

### PR DESCRIPTION
**Pull request by Panquesito7**
Things added/changed:

- Fix spelling and Markdown linter issues in the CoC.
  - CC: @amino19.

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -1,29 +1,30 @@
-# 📰 CODE OF CONDUCT
-   
+# 📰 Code of Conduct
```
Comment (akshatnema) [inline CODE_OF_CONDUCT.md:1]: We can't make the heading in small characters in this page, as on each page we have headings in all Capital letters.

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -1,29 +1,30 @@
...
+
 <!-- 
-! Joshith, make all these points as numbered lists
+TODO: Joshith, make all these points as numbered lists
```
Comment (akshatnema) [inline CODE_OF_CONDUCT.md:4]: That exclamation sign is for the use of the Better Comments extension in VS Code. Better Comments extension allows us to view comments in a more colorful way.

File: CODE_OF_CONDUCT.md [hunk 3]
```diff
@@ -1,29 +1,30 @@
...
+
+8. Indulging in any dishonest means leads to disqualification.
+
+> ### Note
+>
 > Not adhering to the above set of guidelines may lead to disqualification.
-> **strict actions woud be taken on the offendant**.
- 
\ No newline at end of file
+> **strict actions would be taken on the offendant**.
```
Comment (akshatnema) [inline CODE_OF_CONDUCT.md:30]: S should be capital in this line.

**Review by akshatnema — CHANGES_REQUESTED** [review]
@Panquesito7 Should I edit the files in your PR to get it correct and then merge them?

**Comment by Panquesito7** [comment]
> @Panquesito7 Should I edit the files in your PR to get it correct and then merge them?

I'll make the changes. If there are still bugs, feel free to make the changes and merge. 😃

**Review by akshatnema — APPROVED** [review]

**Pull request merged**